geometry_msgs/Point location
float64 cost
